javascript/game.js

```diff
@@ -1,117 +1,69 @@
+$(document).ready(function () {
+	var orange = $('#orange');
+	var cherry = $('#cherry');
+	var banana = $('#banana');
+	var berry = $('#berry');
 
-// random number to try and add too
+	var gameWin;
+	var total;
 
-var gameWin = Math.floor(Math.random() * 120) + 19;
+	var win = 0;
+	var loss = 0;
 
-// display gameWin in #gameNumber div
-$("#gameNumber").html(gameWin);
+	function resetGameNumbers () {
+		gameWin = generateRandomNumber(19, 120);
+		$("#gameNumber").html(gameWin);
 
-// create random numbers for fruit
+		cherry.data('fruit-val', generateRandomNumber(1, 19)); 
+		orange.data('fruit-val', generateRandomNumber(1, 19)); 
+		banana.data('fruit-val', generateRandomNumber(1, 19)); 
+		berry.data('fruit-val', generateRandomNumber(1, 19)); 
 
-var cherry = Math.floor(Math.random() * 19) + 1;
-var orange = Math.floor(Math.random() * 19) + 1;
-var banana = Math.floor(Math.random() * 19) + 1;
-var berry = Math.floor(Math.random() * 19) + 1;
-
-// console log for clarity
-console.log(cherry, orange, banana, berry);
-
-
-
-// var total is will be the value of var totalGuess added together
-var total = 0;
-
-win = 0;
-loss = 0;
-
-function winReset(){
-
-	gameWin = Math.floor(Math.random() * 120) + 19;
-
-	// display gameWin in #gameNumber div
-	$("#gameNumber").html(gameWin);
-
-	// create random numbers for fruit
-
-	var cherry = Math.floor(Math.random() * 19) + 1;
-	var orange = Math.floor(Math.random() * 19) + 1;
-	var banana = Math.floor(Math.random() * 19) + 1;
-	var berry = Math.floor(Math.random() * 19) + 1;
-
-	win++;
-	alert("you win");
-	total = 0;
-	$("#playerScore").html(total);
-	$("#wins").html("Wins: " + win);
-	console.log(cherry, orange, banana, berry);
-
-
-};
-
-function loseReset(){
-	gameWin = Math.floor(Math.random() * 120) + 19;
-
-	// display gameWin in #gameNumber div
-	$("#gameNumber").html(gameWin);
-
-	// create random numbers for fruit
-
-	var cherry = Math.floor(Math.random() * 19) + 1;
-	var orange = Math.floor(Math.random() * 19) + 1;
-	var banana = Math.floor(Math.random() * 19) + 1;
-	var berry = Math.floor(Math.random() * 19) + 1;
-
-	
-	alert("you lose");
-	loss++;
-	total = 0;
-	$("#playerScore").html(total);
-	$("#loss").html("Losses: " + loss);
-	console.log(cherry, orange, banana, berry);
+		total = 0;
+		$("#playerScore").html(total);
+	};
 
-}
+	function generateRandomNumber (min, max) {
```

Comment:
Instead of writing out the funky looking `Math` code to generate a random number a bunch of times, it can be useful to hide that complexity behind a handy helper function. And by accepting arguments the helper function becomes that much more flexible!